.vscode/launch.json

```diff
@@ -98,7 +99,6 @@
       ],
       "console": "integratedTerminal",
       "internalConsoleOptions": "neverOpen",
-      "disableOptimisticBPs": true,
```

Comment:
Asking for my understanding. What is is this property?

Also this is still present in line 80. 
![image](https://github.com/user-attachments/assets/677863b6-4b16-4804-9e57-57288a49b9f2)

Reply:
I started to have it removed and I gave up because it was not clear to me either and I would not like to spend time on this 😄 
The short answer is that initial research did not show it was required, but I would not want to disturb the PR, and this one ended up being removed by mistake.

Reply:
Reverted to add it back.